.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Twister Unit Tests
         working-directory: example-application
         run: |
-          ../zephyr/scripts/twister -p qemu_cortex_m0 --testcase-root ./tests/
+          ../zephyr/scripts/twister -p custom_plank --testsuite-root ./tests/
```

Comment:
doesn't this prevent running test on CI?

Reply:
actually... we should run in integration mode, and set integration platforms in the testcase.yaml (likely native_posix, custom_plank, Qemu).

Reply:
For sure, but this is a hotfix for now.

Reply:
So then why is this change needed?

Reply:
The **needed** change is from testcase-root to testsuite-root.